_includes/digital.html

```diff
@@ -3,9 +3,12 @@
   aria-label="Visit Digital.gov for additional guides and resources"
 >
   <div class="grid-container padding-y-105 display-flex flex-justify flex-align-center">
-    <a href="https://digital.gov/resources"><img class="display-block"
-        src="{{ '/assets/img/digitalgov-logo-small-white.svg' | relative_url }}" alt="Digital.gov Guide" /></a>
-    <a href="https://digital.gov/resources" class="text-white text-no-underline">Explore <i
-        class="fas fa-arrow-right margin-left-1"></i></a>
+    <a href="https://digital.gov/resources">
+      <img class="display-block"
+        src="{{ '/assets/img/digitalgov-logo-small-white.svg' | relative_url }}" alt="Digital.gov Guide" height="36" width="220"/>
+    </a>
+    <a href="https://digital.gov/resources" class="text-white text-no-underline digital-banner__explore-link">Explore
+      <svg class="digital-banner__explore-arrow" xmlns="http://www.w3.org/2000/svg" width="30" height="30" fill="white" viewBox="0 0 24 24"><path d="m12 4-1.41 1.41L16.17 11H4v2h12.17l-5.58 5.59L12 20l8-8z"/></svg>
```

Comment:
Where is this SVG coming from? Could we use a USWDS icon?

Reply:
This `svg` code is from `assets/uswds/img/usa-icons/arrow_forward.svg` and hardcoded into the template because we are ignoring uswds icon folders.